Gordon360/Controllers/RecIM/TeamsController.cs

```diff
@@ -231,8 +231,8 @@ public ActionResult<IEnumerable<TeamExtendedViewModel>> GetTeamInvites()
         /// <param name="response"></param>
         /// <returns>The accepted TeamInviteViewModel</returns>
         [HttpPatch]
-        [Route("{teamID}/invite")]
-        public async Task<ActionResult<ParticipantTeamViewModel?>> AcceptTeamInvite(int teamID, TeamInviteResponseViewModel response)
+        [Route("{teamID}/invite/status")]
+        public async Task<ActionResult<ParticipantTeamViewModel?>> AcceptTeamInvite(int teamID, [FromBody]string response)
```

Comment:
FYI, I don't think you need the `[FromBody]` Attribute here, ASP.NET Core will search the request body for this value by default, according to [the docs](https://learn.microsoft.com/en-us/aspnet/core/mvc/models/model-binding?view=aspnetcore-7.0#sources):

> By default, model binding gets data in the form of key-value pairs from the following sources in an HTTP request:
> 1. Form fields
> 2. The request body (For controllers that have the [ApiController] attribute.)
> 3. Route data
> 4. Query string parameters
> 5. Uploaded files

Reply:
I don't think this is true, I did a quick check to make sure (with intentional error just to show the actual route)
without [FromBody] I'm pretty sure it just goes to default query (I remember seeing this on some of Kevin's team routes)

![image](https://user-images.githubusercontent.com/78386128/219693349-72401ece-1ab1-4f3a-991b-235629674b4d.png)
(without FromBody specification)

Reply:
Ah, that's actually an issue with Swagger. If you were to send a POST request with the response in the body, the API would read that just fine.

Swagger, however, doesn't know where you intend for the data to come from, so it defaults to using query params for simple types like string.

The API doesn't really care where in the request the data is located as long as it can bind successfully - with the one exception that complex data types can't be in the route data or query string because there's no way to encode them in the URL.

Reply:
huh interesting, I'll test it locally on the next run through. (just to confirm, not that I don't believe you) If it works, I'll make a new PR to clean up the FromBody instances we have (if unnecessary)